include/zephyr/posix/unistd.h

```diff
@@ -11,16 +11,217 @@
+#define _POSIX_ARG_MAX			    (4096)
+#define _POSIX_CHILD_MAX		    (25)
+#define _POSIX_DELAYTIMER_MAX		    (32)
+#define _POSIX_HOST_NAME_MAX		    (255)
+#define _POSIX_LINK_MAX			    (8)
+#define _POSIX_LOGIN_NAME_MAX		    (9)
+#define _POSIX_MAX_CANON		    (255)
+#define _POSIX_MAX_INPUT		    (255)
+#define _POSIX_MQ_OPEN_MAX		    CONFIG_MSG_COUNT_MAX
+#define _POSIX_MQ_PRIO_MAX		    (32)
+#define _POSIX_NAME_MAX			    (14)
+#define _POSIX_NGROUPS_MAX		    (8)
+#define _POSIX_OPEN_MAX			    CONFIG_POSIX_MAX_FDS
+#define _POSIX_PATH_MAX			    (256)
+#define _POSIX_PIPE_BUF			    (512)
+#define _POSIX_RE_DUP_MAX		    (255)
+#define _POSIX_RTSIG_MAX		    CONFIG_POSIX_RTSIG_MAX
+#define _POSIX_SEM_NSEMS_MAX		    CONFIG_SEM_NAMELEN_MAX
+#define _POSIX_SEM_VALUE_MAX		    CONFIG_SEM_VALUE_MAX
+#define _POSIX_SIGQUEUE_MAX		    (32)
+#define _POSIX_SSIZE_MAX		    (32767)
+#define _POSIX_SS_REPL_MAX		    (4)
+#define _POSIX_STREAM_MAX		    (8)
+#define _POSIX_SYMLINK_MAX		    (255)
+#define _POSIX_SYMLOOP_MAX		    (8)
+#define _POSIX_THREAD_DESTRUCTOR_ITERATIONS (4)
+#define _POSIX_THREAD_KEYS_MAX		    (128)
+#define _POSIX_THREAD_THREADS_MAX	    (64)
+#define _POSIX_TIMER_MAX		    (32)
+#define _POSIX_TRACE_EVENT_NAME_MAX	    (30)
+#define _POSIX_TRACE_NAME_MAX		    (8)
+#define _POSIX_TRACE_SYS_MAX		    (8)
+#define _POSIX_TRACE_USER_EVENT_MAX	    (32)
+#define _POSIX_TTY_NAME_MAX		    (9)
+#define _POSIX_TZNAME_MAX		    (6)
+#define _POSIX2_BC_BASE_MAX		    (99)
+#define _POSIX2_BC_DIM_MAX		    (2048)
+#define _POSIX2_BC_SCALE_MAX		    (99)
+#define _POSIX2_BC_STRING_MAX		    (1000)
+#define _POSIX2_CHARCLASS_NAME_MAX	    (14)
+#define _POSIX2_COLL_WEIGHTS_MAX	    (2)
+#define _POSIX2_EXPR_NEST_MAX		    (32)
+#define _POSIX2_LINE_MAX		    (2048)
+#define _XOPEN_IOV_MAX			    (16)
+#define _XOPEN_NAME_MAX			    (255)
+#define _XOPEN_PATH_MAX			    (1024)
+
+/* Other invariant values */
+#define NL_LANGMAX (14)
+#define NL_MSGMAX  (32767)
+#define NL_SETMAX  (255)
+#define NL_TEXTMAX (_POSIX2_LINE_MAX)
+#define NZERO	   (20)
+
+/* Runtime invariant values */
+#define AIO_LISTIO_MAX		      _POSIX_AIO_LISTIO_MAX
+#define AIO_MAX			      _POSIX_AIO_MAX
+#define AIO_PRIO_DELTA_MAX	      (0)
+#define DELAYTIMER_MAX		      _POSIX_DELAYTIMER_MAX
+#define HOST_NAME_MAX		      COND_CODE_1(CONFIG_NETWORKING,	  \
+						  (NET_HOSTNAME_MAX_LEN), \
+						  (_POSIX_HOST_NAME_MAX))
+#define LOGIN_NAME_MAX		      _POSIX_LOGIN_NAME_MAX
+#define MQ_OPEN_MAX		      _POSIX_MQ_OPEN_MAX
+#define MQ_PRIO_MAX		      _POSIX_MQ_PRIO_MAX
+
+#ifndef PAGE_SIZE
+#define PAGE_SIZE		      (1)
+#endif
+
+#ifndef PAGESIZE
+#define PAGESIZE		      PAGE_SIZE
+#endif
+
+#define PTHREAD_DESTRUCTOR_ITERATIONS _POSIX_THREAD_DESTRUCTOR_ITERATIONS
+#define PTHREAD_KEYS_MAX	      COND_CODE_1(CONFIG_PTHREAD_IPC,		  \
+						  (CONFIG_MAX_PTHREAD_KEY_COUNT), \
+						  (_POSIX_THREAD_KEYS_MAX))
+#define PTHREAD_THREADS_MAX	      COND_CODE_1(CONFIG_PTHREAD_IPC, CONFIG_MAX_PTHREAD_COUNT, 0)
```

Comment:
`COND_CODE_1` requires that it's arguments have parenthesis

```suggestion
#define PTHREAD_THREADS_MAX	      COND_CODE_1(CONFIG_PTHREAD_IPC, (CONFIG_MAX_PTHREAD_COUNT), (0))
```